rust/Cargo.toml

```diff
@@ -91,8 +91,12 @@ elliptic-curve = "0.12.3"
 getrandom = { version = "0.2", features = ["js"] }
 hex = "0.4.3"
 hpl-interface = "=0.0.6-rc3"
+# Must be the same version used by Tonic
```

Comment:
not sure if there's a smarter way of doing this. iirc "*" didn't satisfy this, probably because there are so many different packages that use http